se/se_epub_build.py

```diff
@@ -239,10 +239,11 @@ def build(self, metadata_xhtml: str, metadata_tree: se.easy_xml.EasyXmlTree, run
 								while selector_to_simplify in selector:
 									# Potentially the pseudoclass we’ll simplify isn’t at the end of the selector,
 									# so we need to temporarily remove the trailing part to target the right elements.
-									target_element_selector = ''.join(selector.partition(selector_to_simplify)[0:2])
+									split_selector = regex.split(f"({selector_to_simplify}(\(.*\))?)", selector, 1)
+									target_element_selector = ''.join(split_selector[0:2])
 
-									replacement_class = selector_to_simplify.replace(":", "")
-									selector = selector.replace(selector_to_simplify, "." + replacement_class, 1)
+									replacement_class = split_selector[1].replace(":", "").replace("(", "-").replace("n-", "n-minus-").replace("n+", "n-plus-").replace(")", "")
```

Comment:
Do we have a place to put small helper functions? It would be nice to make this replace() sequence a function since there is an identical copy in formatting.py. That way they don't get out of sync if the pseudo-class list changes again.

Reply:
We don’t, but if @acabal think’s it’s useful let’s add one. Incidentally, these aren’t exactly the same as one prepends a full stop. Trivial to work around, but nb if we come back to this.

Reply:
We have space for project-wide helper functions in `__init__.py`, but if we're just saving a few lines then it might not be worth it to add complexity.

Reply:
It’s not so much as saving lines, as centralising logic. But the logic here isn’t particularly complex and is intrinsically linked to the other class generation, so it’s be difficult for someone to touch this without also wanting to touch the other place. So 🤷‍♂️, I’m happy with it as it is, but if you think it should be moved I’ll change it.